app/controllers/ems_common.rb

```diff
@@ -678,6 +699,38 @@ def refreshemss
     end
   end
 
+  def call_ems_disable(emss)
+    process_emss(emss, "disable_ems") unless emss.empty?
+    return if @flash_array.present?
+
+    add_flash(n_("Disable initiated for %{count} %{model} from the %{product} Database",
+                 "Disable initiated for %{count} %{models} from the %{product} Database", emss.length) %
```

Comment:
I think you only allowed disable for individual managers, right? 
That's probably a good idea. You would have a weird behavior for more then one, if some are already disabled.
(some would be disabled and some enabled)